### PR TITLE
Bug fixing for 3.2.0

### PR DESCRIPTION
**Pull request by emcek**
## Description
* bug fixing
* release stabilization (@sleighzy, @emcek)

## Todo
- [x] `key_down` for G-Keys issue
- [x] test and fix `FA18Chornet`
- [x] test and fix `Ka50` & `Ka503`
- [x] test and fix `Mi8MT`
- [x] test and fix `Mi24P`
- [x] test and fix `F16C50`
- [x] test and fix `F15ESE`
- [x] test and fix `AH64DBLKII`
- [x] test and fix `A10C`
- [x] test and fix `A10C2`
- [x] test and fix `F14A135GR` & `F14B`

### CODE REVIEW

**Comment by emcek** [comment]
@sleighzy test and fix here

File: tests/test_logitech.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ def test_keyboard_base_basic_check(keyboard_base):
 
     assert str(keyboard_base) == 'KeyboardManager: 160x43'
     logitech_repr = repr(keyboard_base)
-    data = ('parser', 'ProtocolParser', 'plane_name', 'plane_detected', 'lcdbutton_pressed', 'buttons',
```
Comment (sleighzy) [inline tests/test_logitech.py:14]: Looks like I hadn't reverted this change in the test. Doesn't affect anything so can be readded in a later release.

**Review by sleighzy — APPROVED** [review]
LGTM.

File: tests/test_logitech.py (hunk 1 above)
Comment (emcek) [reply tests/test_logitech.py:14]: I will add in master

**Pull request merged**